config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  # root '/greetings'
+  resources :greetings, only: [:index] do
```

Comment:
- [ ] Please make sure to set the root route as the greeting index page so users will be able to see the greetings when they run the app.
  - You can consider using `root 'greetings#index'`.

Reply:
well resolved! ✔️